exporter/otlp/lib/opentelemetry/exporter/otlp/version.rb

```diff
@@ -8,7 +8,7 @@ module OpenTelemetry
   module Exporter
     module OTLP
       ## Current OpenTelemetry OTLP exporter version
-      VERSION = '0.24.0'
+      VERSION = '0.24.1'
```

Comment:
I think our release machinery will handle bumping this

Reply:
Done!